Gulpfile.coffee

```diff
@@ -92,6 +92,12 @@ gulp.task 'tdd', ['build'],  (done) ->
   return # Since this is async
 
 gulp.task 'buildJS', ['cleanJS'], -> build(false)
+gulp.task 'watchJS', ['cleanJS'], (done)->
+  build(true)
+  buildTests(true).on 'end', ->
+    config =
+      configFile: __dirname + '/test/karma.config.coffee'
+    karma.server.start(config, done)
```

Comment:
Could you pull this bit out into a shared function since test, tdd, and watchJS probably all have similar code (but different args to build/buildTests)